src/components/Image/imagePropTypes.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+import stylePropTypes from '../../styles/stylePropTypes';
+import RESIZE_MODES from './resizeModes';
+
+const propTypes = {
+    /** Styles for the Image */
+    style: stylePropTypes,
+
+    /** The static asset or URI source of the image */
+    source: PropTypes.oneOfType([
+        PropTypes.number.isRequired,
```

Comment:
Just wondering, is `isRequired` here? I don't see it after the next `PropTypes.shape` so I am guess it's either needed on both or neither?

Reply:
Yes good catch - its required to either be an object with a URI or a `require()` (a number) so I've set it for both now :)